db-backup-files/2-patternatlas-data-inserts.sql

```diff
@@ -263,29 +264,29 @@ INSERT INTO public.pattern VALUES ('678a2e5a-697f-4416-afe6-fd18bc935bdc', 'Hybr
olumn vector\n\\left( \\begin{matrix}#1\\end{matrix}\\right)\n}\n\\newcommand{\\state}[1]{{\\left| #1 \\right>}} \\state{0}^{\\otimes n}  \\state{0}^{\\otimes m}  \\xrightarrow{H^{\\otimes n} \\otimes I} \\left(  \\frac{1}{\\sqrt{2^n}} \\sum_x \\state{x} \\right) \\otimes \\state{0}^{\\otimes m} $$\n\n\n$$\n\\newcommand{\\colVec}[1]{% inline column vector\n\\bigl( \\begin{smallmatrix}#1\\end{smallmatrix}\\bigr)\n}\n\\newcommand{\\bigColVec}[1]{% inline column vector\n\\left( \\begin{matrix}#1\\end{matrix}\\right)\n}\n\\newcommand{\\state}[1]{{\\left| #1 \\right>}}\\xrightarrow{U_f} \\frac{1}{\\sqrt{2^n}} \\sum_x \\state{x}\\state{f(x)} $$\nIf $f$ is an one-dimensional function (e.g., for solving a decision problem), the register is initialized as $\\newcommand{\\state}[1]{{\\left| #1 \\right>}} \\state{0}^{\\otimes n} \\state{1}$.  \nAfterward, the Hadamard operation is performed on the complete register, i.e. the operator $H^{\\otimes n+1}$ is used.  \nApplication of $U_f$ results in the following state:\n\n\n$$\n\\newcommand{\\colVec}[1]{% inline column vector\n\\bigl( \\begin{smallmatrix}#1\\end{smallmatrix}\\bigr)\n}\n\\newcommand{\\bigColVec}[1]{% inline column vector\n\\left( \\begin{matrix}#1\\end{matrix}\\right)\n}\n\\newcommand{\\state}[1]{{\\left| #1 \\right>}}  \\state{0}^{\\otimes n}  \\state{1}  \\xrightarrow{H^{\\otimes n} \\otimes H} \\left(  \\frac{1}{\\sqrt{2^n}} \\sum_x \\state{x} \\right) \\otimes \\state{-}$$\n\n\n$$\n\\newcommand{\\colVec}[1]{% inline column vector\n\\bigl( \\begin{smallmatrix}#1\\end{smallmatrix}\\bigr)\n}\n\\newcommand{\\bigColVec}[1]{% inline column vector\n\\left( \\begin{matrix}#1\\end{matrix}\\right)\n}\n\\newcommand{\\state}[1]{{\\left| #1 \\right>}}\\xrightarrow{U_f} \\left(  \\frac{1}{\\sqrt{2^n}} \\sum_{x=0}^{2^n-1} (-1)^{f(x)} \\state{x}  \\right) \\otimes \\state{-} $$", "Variants": "Enter your input for this section here.", "Known Uses": "Many algorithms, including the algorithms of Deutsch, Deutsch-Jozsa, Grover, and Shor make use of function tables.", "Related Patterns": "This pattern uses [Initialization](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/312bc9d3-26c0-40ae-b90b-56effd136c0d) and [Uniform Superposition](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/2229a430-fe92-4411-9d72-d10dd1d8da14) to prepare the initial state of the quantum register. \nFunction tables can be further generalized as [Amplitude Amplification](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/96b4d28a-a5ce-4c96-85df-d42587b13c57). \nThe computation of the function table is done by an [Oracle](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/1cc7e9d6-ab37-412e-8afa-604a25de296e). \nOften, [Uncompute](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/d4f7c247-e2bb-4301-ad06-f758fa58f2dc) is required to dissolve the created entanglement for subsequent steps of the algorithm."}', 'af7780d5-1f97-4536-8da7-4194b093ab1d', 'https://link.springer.com/chapter/10.1007/978-3-030-14082-3_19');
```

Comment:
in section Result, first '' is used instead of "

Reply:
further, in Relations to other Patterns the isRelatedTo is presenting strange rendering--> "Function Table: Applying U_{f}U f ​ Applying U_{f}U f ​ Applying the unitary ..."
same case also in pattern Creating Entanglement